crates/bcr-ebill-core/src/nostr_contact.rs

```diff
@@ -3,16 +3,21 @@ use std::collections::BTreeSet;
 use secp256k1::SecretKey;
 use serde::{Deserialize, Serialize};
 
-use crate::{ValidationError, contact::Contact};
+use crate::{
+    NodeId, ValidationError,
+    contact::{Contact, ContactType},
+};
 
 /// Make key type clear
 pub type NostrPublicKey = nostr::key::PublicKey;
 
 /// Data we need to communicate with a Nostr contact.
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct NostrContact {
-    /// Our node id. This is the node id and acts as the primary key.
+    /// Our node id npub. This is the node id and acts as the primary key.
```

Comment:
This sounds a bit strange 😅 
maybe something like `Our npub - this is the node identifier on the Nostr network and acts as the primary key`